src/main/java/com/northcoders/makemydayapi/controller/ActivityController.java

```diff
@@ -1,4 +1,51 @@
 package com.northcoders.makemydayapi.controller;
 
+
+import com.northcoders.makemydayapi.model.Activity;
+import com.northcoders.makemydayapi.service.GooglePlacesService;
+import com.northcoders.makemydayapi.service.SkiddleService;
+import com.northcoders.makemydayapi.service.TicketmasterService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/v1/activities")
 public class ActivityController {
+
+    @Autowired
+    private GooglePlacesService googlePlacesService;
+
+    @Autowired
+    private SkiddleService skiddleService;
+
+    @Autowired
+    private TicketmasterService ticketmasterService;
+
+//    google places activities endpoint
+    @GetMapping("/google-places")
+    public ResponseEntity<List<Activity>> getGooglePlacesActivities() {

```

Comment:
It's probably a placeholder.

We can replace it with `geoapify` later, after merging its corresponding branch into `main`.